test.sh

```diff
@@ -0,0 +1,2 @@
+python -m unittest common.tests.util.tests
```

Comment:
This shell file is missing a shebang, depending on your shell `./test.sh` may not do what you think.

Please add `#!/bin/sh` as the very first line